plugins/web/opentelemetry-instrumentation-user-interaction/src/types.ts

```diff
@@ -14,7 +14,28 @@
  * limitations under the License.
  */
 
-import * as types from '@opentelemetry/api';
+import { HrTime, Span } from '@opentelemetry/api';
+import { InstrumentationConfig } from '@opentelemetry/instrumentation';
+
+export interface UserInteractionInstrumentationConfig
+  extends InstrumentationConfig {
+  /**
+   * List of events to instrument (like 'mousedown', 'touchend', 'play' etc).
+   * By default only 'click' event is instrumented.
+   */
+  eventTypes?: string[];
```

Comment:
if that is possible I would create a typed event here - so that you can have type validation and allows only certain events (click, double clicks etc.)